### PR TITLE
chore Base cleanup

### PR DESCRIPTION
**Pull request by jtnord**
Cleanup the base builder files.

* The main files are auto generated and the ones in git are very old and cause confusion.
* The slim image was using old versions, so now generate it on the fly with the auto updated common versions.
* The `Dockerfile.php` was not being used and was usign old versions so just remove it.

### CODE REVIEW

File: builder-base/Dockerfile.swift [hunk 1]
```diff
@@ -1,126 +0,0 @@
-FROM swift:4.0.3
```
Comment (jtnord) [inline builder-base/Dockerfile.swift]: is generated [here](https://github.com/jenkins-x/jenkins-x-builders/pull/131/files#diff-e091688d80187842c1b91dcc19485fbaR105)

File: builder-base/Dockerfile.ruby [hunk 2]
```diff
@@ -1,126 +0,0 @@
-FROM ruby:2.5.1
```
Comment (jtnord) [inline builder-base/Dockerfile.ruby]: is generated [here](https://github.com/jenkins-x/jenkins-x-builders/pull/131/files#diff-e091688d80187842c1b91dcc19485fbaR104)

File: builder-base/Dockerfile.php [hunk 3]
```diff
@@ -1,86 +0,0 @@
-FROM php:7.2.5
```
Comment (jtnord) [inline builder-base/Dockerfile.php]: was neither built nor updated.

File: builder-base/Dockerfile [hunk 4]
```diff
@@ -1,140 +0,0 @@
-FROM centos:7
```
Comment (jtnord) [inline builder-base/Dockerfile]: is generated [here](https://github.com/jenkins-x/jenkins-x-builders/pull/131/files#diff-e091688d80187842c1b91dcc19485fbaR40)

File: builder-base/Dockerfile.slim.commands [hunk 5]
```diff
@@ -1,7 +1,5 @@
-FROM centos:7
```
Comment (jtnord) [inline builder-base/Dockerfile.slim.commands]: now generated [here](https://github.com/jenkins-x/jenkins-x-builders/pull/131/files#diff-e091688d80187842c1b91dcc19485fbaR52)

**Comment by vbehar** [comment]
@jtnord it would be nice to merge this PR. it's confusing to see all these dockerfiles when first coming to this repository

**Pull request merged**